lisa/tools/gpu_drivers.py

```diff
@@ -517,6 +530,71 @@ def _install_cuda_ubuntu(self) -> None:
 
         self._log.info("Successfully installed CUDA driver for Ubuntu")
 
+    def _install_cuda_debian(self) -> None:
```

Comment:
Try to create a common method that works for both Debian and Ubuntu, avoiding duplicated code.

Reply:
updated